nodecontrol/algodControl.go

```diff
@@ -85,7 +85,11 @@ func (nc NodeController) ServerURL() (url.URL, error) {
 	if err != nil {
 		return url.URL{}, err
 	}
-	return url.URL{Scheme: "http", Host: addr}, nil
+	scheme := "http"
+	if util.IsGlobalUnicast(addr) {
```

Comment:
not sure if using `IsGlobalUnicast` is a right thing for https vs http connection scheme. I guess a proper solution should either try http and then https, or algod.net might include an optional scheme prefix.

Reply:
I like the last solution so we don't have to try to be clever about it. If they want https to work, specify the url as https in the algod.net